buffalo/cmd/new.go

```diff
@@ -1,11 +1,12 @@
 package cmd
 
 import (
-	"fmt"
 	"os"
 	"os/user"
 	"path/filepath"
 
+	"github.com/Sirupsen/logrus"
```

Comment:
The repo is now lower case `sirupsen`. Can you please undo these import changes?